content/https/https-6/contents.lr

```diff
@@ -0,0 +1,17 @@
+_model: question
+---
+title: Why Tor Browser doesn't deploy DoH?
+---
+seo_slug: reason-tor-not-deploy-doh
+---
+description:
+There are two main reasons:
+DoH is centralized and there are not many diverse DoH servers today.

```

Comment:
I'm not sure this is entirely accurate. Sure, "there are not many diverse DoH servers today" but it's not completely true "DoH is centralized", at least not by design. We could see a situation where there are a wide variety of DNS providers offering DoH service and you'd be free to pick the one you feel better respects your privacy, for example.